codespell_lib/data/dictionary.txt

```diff
@@ -9570,6 +9580,7 @@ guarunty->guaranty
 guassian->gaussian
 Guatamala->Guatemala
 Guatamalan->Guatemalan
+gude->good, guide,
```

Comment:
I suspect guide should probably be the first suggestion.

Also add `gud->good`

Reply:
c6c6b2d